package.json

```diff
@@ -20,6 +20,8 @@
     "test:scripts": "lerna run test:scripts --stream",
     "test:scripts:cross-platform": "lerna run test:scripts:cross-platform --stream",
     "test:e2e": "lerna run test:e2e --ignore @mendix/custom-widgets-utils-internal --stream --concurrency 1 --no-bail",
+    "test:e2e:local:ios": "detox test --configuration ios.simulator.developerapp",
```

Comment:
It's safer to use the following command here: `lerna run test:e2e:local:ios --stream --concurrency 1 --scope "*-native"`

This will run `npm run run test:e2e:local:ios` in every `*-native` folder.

Reply:
Alright! Changed as needed.